lib/gitspindle/github.py

```diff
@@ -1093,14 +1090,15 @@ def protect(self, opts):
         """[--enforcement-level=<level>] [--contexts=<contexts>] <branch>
            Protect a branch against deletions, force-pushes and failed status checks"""
         repo = self.repository(opts)
-        repo.branch(opts['<branch>']).protect(enforcement_level=opts['--enforcement-level'], contexts=(opts['--contexts'] or '').split(','))
+        repo.branch(opts['<branch>']).protect(enforcement=opts['--enforcement-level'],
+                                              status_checks=(opts['--contexts'] or '').split(','))
```

Comment:
Let's rename the `--context` and `--enforcement` parameters as well.

Reply:
done